examples/angular/src/app/components/contentlets/contentlets.component.ts

```diff
@@ -1,19 +1,18 @@
 import { Component, Input } from '@angular/core';
 import { Contentlet } from '@dotcms/client/src/lib/client/content/shared/types';
 import { GenericContentlet } from '../../utils';
-import { environment } from '../../../environments/environment';
-import { NgOptimizedImage } from '@angular/common';
+import { DatePipe, NgOptimizedImage } from '@angular/common';
 
 @Component({
   selector: 'app-contentlets',
   standalone: true,
-  imports: [NgOptimizedImage],
+  imports: [NgOptimizedImage, DatePipe],
   template: `<ul class="flex flex-col gap-7">
     @for(contentlet of contentlets; track contentlet.identifier) {
     <li class="flex gap-7 min-h-16">
       <a class="min-w-32 relative" [href]="contentlet.urlMap ?? contentlet.url">
         <img
-          [ngSrc]="getImageUrl(contentlet)"
+          [ngSrc]="contentlet.image"
```

Comment:
@KevinDavilaDotCMS In the previous example the `language_id` was part of the image, in this case an example is missing to send this param?, like this:

```html
<img  [ngSrc]="contentlet.image"  [loaderParams]="{contentlet.languageId || 1}"
```

Reply:
You right
The image can be rendered without language id, but for the example i added a default languageId=1 in ImageLoaderConfig and the line you propose
[loaderParams]="{"languageId": contentlet.languageId || 1}"
to provide the user an example of loader Params :D